samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -231,10 +230,11 @@ static const char link_rsrp_usage_str[] =
 	"  -u, --unsubscribe,  Unsubscribe for RSRP info\n";
 
 static const char link_ncellmeas_usage_str[] =
-	"Usage: link ncellmeas --single | --cancel\n"
+	"Usage: link ncellmeas --single | --continuous |--cancel\n"
 	"Options:\n"
-	"      --single, Start a neighbor cell measurement and report result\n"
-	"      --cancel, Cancel/Stop neighbor cell measurement if still on going\n";
+	"      --single,     Start a neighbor cell measurement and report result\n"
+	"      --continuous, Start continuous neighbor cell measurement mode and report result\n"
+	"      --cancel,     Cancel/Stop neighbor cell measurement if still ongoing\n";
```

Comment:
I know this is older stuff but seeing that stop is used in the entire code, should we rename `--cancel` to `--stop`?

Reply:
stop was one of the options, however this was discussed and the decision earlier was that this to be cancel. However, fixed help test from main level to be cancel.